easybuild/easyconfigs/c/cuDNN/cuDNN-7.5.0-cuda-10.0.eb

```diff
@@ -0,0 +1,28 @@
+# @author: gppezzi, marcelsc
+
+easyblock = 'Tarball'
+
+name = 'cuDNN'
+version = '7.5.0'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """Nvidias deep neural network library."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+cudamajor = '10.0'
+
+versionsuffix = '-cuda-%s' % cudamajor
+
+dependencies = [
+    ('craype-accel-nvidia60', EXTERNAL_MODULE),
```

Comment:
@rsarm Do you really need `craype-accel-nvidia60` as dependency? Or is it enough `cudatoolkit` as a builddependency?

Reply:
cuDNN is only available as binaries. I assumed that when the module is loaded, it would need `cudatoolkit` but now that you mention it, it I do `ldd` to the cuDNN binaries, I see that it doesn't refer to `cudatoolkit`. Should I remove `craype-accel-nvidia60`?

Reply:
Yes, please remove it if it is not necessary